src/ansys/additive/core/additive.py

```diff
@@ -159,7 +160,10 @@ def __init__(
         self._servers = Additive._connect_to_servers(
             server_connections, host, port, nservers, product_version, self._log, linux_install_path
         )
-        self._nsims_per_server = nsims_per_server
+
+        initial_settings = {"NumConcurrentSims": str(nsims_per_server)}
+        print(self.apply_server_settings(initial_settings))
```

Comment:
Let's use `LOG.info()` instead of `print`.

Reply:
Fixed.